packages/react-components/react-search/docs/Spec.md

```diff
@@ -2,23 +2,79 @@
 
 ## Background
 
-_Description and use cases of this component_
+`SearchBox` is a text input field abstracting `<input type="search" />`.
 
 ## Prior Art
 
-_Include background research done for this component_
+- [Ant Design](https://ant.design/components/input)
+- [Carbon](https://carbondesignsystem.com/components/search/usage/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/input/)
+- [Material UI](https://mui.com/material-ui/react-text-field/)
+- [Spectrum](https://react-spectrum.adobe.com/react-spectrum/SearchField.html)
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### Comparison of v8 and v0
+
+#### v8
+
+The v8 SearchBox component is implemented as a text input field that lacks a type attribute.
+
+The v8 component supports:
+
+- a customizable leading icon, with the default being a search icon
+- hiding the leading icon when the search box is in focus by default, to allow the search content to span the entire width of the search box
+- animation of the disappearance and reappearance of the leading icon, sliding in and out from the left side of the search box
+- a clear button that appears at the end of the search bar, present in the tab order, when the component is in focus and the search box is not empty
+
+The v8 component spans the entire width of its parent component. There are only three visual variants of the v8 component: default and underlined. The v8 component can be disabled.
+
+[Documentation for v8 SearchBox](https://developer.microsoft.com/en-us/fluentui#/controls/web/searchbox)
+
+```
+<SearchBox
+    ariaLabel="SearchBox"
+    placeholder="Search"
+    onSearch={newValue => console.log('value is ' + newValue)}
+/>
+```
+
+#### v0
+
+Because v0 does not have a designated search component, search functionalities fall upon the Input component. The v0 Input component is implemented as a text input field with `type="text"`. If the input is search, the input will include `role="search"`.
+
+The v0 component supports:
+
+- an icon positioned at either the beginning or the end of the input
+- if the input is clearable, the clear icon will replace the custom icon when there is content in the input box and it is in focus
+- being used inline with text
+- input labels, which can be placed outside, inline, or inside of the component
+
+The v0 component has a fixed width that can be changed to span the length of its parent component. Visual variants of the v0 component can be controlled using props. The v0 component can be disabled.
+
+[Documentation for v0 Input](https://fluentsite.z22.web.core.windows.net/0.59.0/components/input/definition)
+
+```
+<Input
+    icon={<SearchIcon />}
+    iconPosition="start"
+    label="Search"
+    placeholder="Search..."
+/>
+```
 
 ## Sample Code
 
 _Provide some representative example code that uses the proposed API for the component_
```

Comment:
There really should be sample code showing the API of the search box for the primary scenarios